### ISSUE TITLE
ExceptionInInitializerError at net.time4j.android.ApplicationStarter$1.run

### ISSUE THREAD

**Issue by fnlctrl** [MenoData/Time4J#802]
Version: v3.44-2018e

Seeing 50 reports of crashes after updating from v3.42-2018e to v3.44-2018e, in the hope of getting rid of the IllegalArgumentException crash (https://github.com/MenoData/Time4J/issues/790). The IllegalArgumentException seems gone but there are more ExceptionInInitializerError crashes..

Crash log from Google Play 
![image](https://user-images.githubusercontent.com/5557143/42956335-532eaf38-8bb2-11e8-8165-b6ae30e29dd5.png)

```
java.lang.ExceptionInInitializerError: 
  at net.time4j.tz.Timezone.ofSystem (Timezone.java)
  at net.time4j.android.ApplicationStarter$1.run (ApplicationStarter.java)
  at java.lang.Thread.run (Thread.java:776)
Caused by: java.lang.NullPointerException: 
  at net.time4j.tz.Timezone.access$602 (Timezone.java)
  or                     .access$902 (Timezone.java)
  or                     .compareTZDB (Timezone.java)
  or                     .fillEtcetera (Timezone.java)
  or                     .getDisplayName (Timezone.java)
  or                     .getID (Timezone.java)
  or                     .getOffset (Timezone.java)
  or                     .getPreferredIDs (Timezone.java)
  or                     .getTZ (Timezone.java)
  or                     .getZoneByAlias (Timezone.java)
  or                     .loadPredefined (Timezone.java)
  or                     .of (Timezone.java)
  or                     .resolve (Timezone.java)
  or                     .with (Timezone.java)
  at net.time4j.tz.Timezone.getDefaultTZ (Timezone.java)
  or                     .getProvider (Timezone.java)
  at net.time4j.tz.Timezone.<clinit> (Timezone.java)
  at net.time4j.tz.Timezone.ofSystem (Timezone.java)
  at net.time4j.android.ApplicationStarter$1.run (ApplicationStarter.java)
  at java.lang.Thread.run (Thread.java:776)
```

**Comment by MenoData** [comment]
The stacktrace was hard to read due to the or-lines but I think now the reason is following (inside class `Timezone`): 

- `getDefaultTZ()` is called before assigning the constant `SYSTEM_TZ_ORIGINAL`
- then `getTZ(...)` is called
- then `Timezone.ofSystem()` is called accessing the still uninitialized constant (which is actually null)

**Comment by MenoData** [comment]
I have now released v3.44.1-2018e (Time4A) which should fix your problem.

**Comment by fnlctrl** [comment]
Thanks for the quick fix!
